change/office-ui-fabric-react-2019-12-24-12-39-09-user-v-satgar-addedRoleAsMenuItem.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Added role as menuitem to the child div",
+  "packageName": "office-ui-fabric-react",
+  "email": "email not defined",
```

Comment:
Can you update this file to have your email? And set it in git config for the future? (run `git config --global user.email you@microsoft.com`)

Reply:
Added email address